### DISCUSSION TITLE
s3 resource

### DISCUSSION THREAD

**Discussion by xuexh0816** [apache/doris#11470] [A - General / Q&A]
errCode = 2, detailMessage = Unsupported resource type: s3
文档中有创建s3的示例，但是我执行之后显示不支持s3 
是用的1.1.1版本

是现在还没有这个功能吗？

我是想吧冷数据存在s3上

**Comment by dataalive** [comment]
you mean backup to S3? create resource fail ?

**Comment by zy-kkk** [comment]
你好，s3作为resource未在1.1.x版本支持，会在即将发布的1.2支持 issue:#8807;pr:#8808
